dataflow/src/main/java/org/checkerframework/dataflow/livevariable/LiveVarNode.java

```diff
@@ -1,12 +1,10 @@
 package org.checkerframework.dataflow.livevariable;
 
 import org.checkerframework.checker.nullness.qual.Nullable;
-import org.checkerframework.dataflow.analysis.AbstractValue;
 import org.checkerframework.dataflow.cfg.node.Node;
-import org.checkerframework.javacutil.BugInCF;
 
-/** A live variable (which is represented by a node) wrapper turning node into abstract value. */
-public class LiveVarValue implements AbstractValue<LiveVarValue> {
+/** A LiveVarNode contains a CFG node, which can only be a LocalVariableNode or FieldAccessNode. */
```

Comment:
Say something about why it is needed, e.g. the special `equals` implementation.

Reply:
I added the role of `LiveVarNode ` and functionality of `equals` at here, but I am not sure if I understand you correctly. Can you check again for the modified version @wmdietl

Reply:
continued the discussion in another thread.